### PR TITLE
Add overview page

### PR DESCRIPTION
**Pull request by ekoleda-codaio**
A place to provide the 10K foot view of Packs. [Staged here](https://coda-packs-sdk--1903.com.readthedocs.build/en/1903/guides/overview/). CC @punit-codaio.

### CODE REVIEW

File: docs/guides/overview.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    <img src="../../images/overview_action.png" srcset="../../images/overview_action_2x.png 2x" alt="A custom action in a button">
+
+    [Learn more][actions]{ .md-button }
+
+=== "Column formats"
+
+    Packs can add new [column formats][help_format], which control how values are displayed within a table. They can simply reformat the data or enrich it with related information from an external source.
+
+    The screenshot below shows a column format from the Google Calendar Pack that displays rich information about an event provided it's URL.
```
Comment (jonathan-codaio) [inline docs/guides/overview.md:45]: `its` 😄

File: docs/guides/overview.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Anyone with a Coda account is able to build a Pack, and getting started is as easy as clicking the "Create a Pack" button in the app. Our Pack Studio comes bundled with all the tools you'll need to write, build, and deploy a Pack, all without leaving your browser. And for more advanced developers we provide a command-line tool, allowing you to use your favorite IDE, tooling, and version control system.
+
+You will need to have a basic understanding of JavaScript or TypeScript to be successful, so if you're new to coding an [online JavaScript course][codecademy] might be a good place to start. We also have a large library of examples and templates built into the Pack Studio, plus a [friendly community][community] of fellow Pack makers that can help you along the way.
+
+[Build your first Pack][quickstart_web]{ .md-button }
+
+
+## How do Packs work?
+
+Each Pack is it's own [serverless application][wikipedia_serverless], fully managed and run by Coda. All you have to do is write the code and we'll take care of the rest.
```
Comment (jonathan-codaio) [inline docs/guides/overview.md:73]: Ditto

**Review by jonathan-codaio — APPROVED** [review]

File: docs/guides/overview.md (hunk 1 above)
Comment (ekoleda-codaio) [reply docs/guides/overview.md]: Done.

File: docs/guides/overview.md (hunk 2 above)
Comment (ekoleda-codaio) [reply docs/guides/overview.md]: Done

**Pull request merged**